chemked/validation.py

```diff
@@ -174,26 +174,26 @@ def _validate_isvalid_reference(self, isvalid_reference, field, value):
                     self._error(field, 'missing author ' +
                                 ' '.join([author['given'], author['family']])
                                 )
-                else:
-                    # validate ORCID
-                    orcid = author.get('ORCID')
-                    if orcid:
-                        orcid = orcid[orcid.rfind('/') + 1 :]
-                        if author_match['ORCID'] != orcid:
-                            self._error(
-                                field, 'author ' +
-                                ' '.join([author['given'], author['family']]) +
-                                ' ORCID should be ' + orcid
-                                )
-                    elif author_match.get('ORCID'):
-                        # still validate if present in file
-                        res = orcid_api.search_public('orcid:' +
-                                                      author_match['ORCID']
-                                                      )
-                        if res['orcid-search-results']['num-found'] == 0:
-                            self._error(field, 'ORCID incorrect for ' +
-                                        author_match['name']
-                                        )
+                # else:
```

Comment:
Before separating the ORCID validator off the DOI validator, this function checked if the ORCID in reference metadata matched the given ORCID. For older articles this won't happen much, but now with article submissions I think publishers are pushing people to get ORCIDs and associate them with papers (and rightfully so).

If an author has an ORCID associated with the reference, we can (a) quickly check that it matches the given ORCID, if one is given; or (b) suggest the ORCID be added if it is not given.